vital/algo/resection_camera.cxx

```diff
@@ -0,0 +1,86 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/**
+ * \file
+ * \brief resection_camera instantiation
+ */
+
+#include <vital/algo/algorithm.txx>
+#include <vital/algo/resection_camera.h>
+
+/// \cond DoxygenSuppress
+INSTANTIATE_ALGORITHM_DEF( kwiver::vital::algo::resection_camera );
+/// \endcond
+
+using namespace std;
+
+namespace kwiver {
+
+namespace vital {
+
+namespace algo {
+
+resection_camera
+::resection_camera()
+{
+  attach_logger( "algo.resection_camera" );
+}
+
+camera_perspective_sptr
+resection_camera
+::resection( frame_id_t frmID,
+             landmark_map_sptr landmarks,
+             feature_track_set_sptr tracks,
+             unsigned width, unsigned height ) const
+{
+  vector< vector_3d > pts_3d; // world points
+  vector< vector_2d > pts_projs; // corresponding image points
+  for( auto const& track : tracks->tracks() )
+  {
+    auto lm_id = track->id();
+    auto lm_it = landmarks->landmarks().find( lm_id );
+    pts_3d.push_back( lm_it->second->loc() );
+    auto const fts =
+      dynamic_pointer_cast< feature_track_state >( *track->find( frmID ) );
+    pts_projs.push_back( fts->feature->loc() );
+  }
```

Comment:
Instead of duplicating this code, please refactor to call the overload that takes tracks, landmarks and intrinsics.

Reply:
addressed